platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/AjaxEngine.ts

```diff
@@ -0,0 +1,166 @@
+// class definition of DOMContext.
+class AjaxEngine {
+	private readonly nodes = new Map<number, Node>();
+	private zIndex = 100;
+
+	public allocateZIndex() {
+		this.zIndex += 1;
+		return '' + this.zIndex;
+	}
+
+	public setMaximumZIndex(node: HTMLElement) {
+		if(node.style.zIndex != '' + this.zIndex) {
+			node.style.zIndex = this.allocateZIndex();
+		}
+	}
+
+	public getNode(nodeId: number) {
+		return this.nodes.get(nodeId) as Node;
+	}
+
+	public getElement(nodeId: number) {
+		return this.nodes.get(nodeId) as HTMLElement;
+	}
+
+	// initializes the head object of this document
```

Comment:
I think we should use a homogenous comment style across all `.ts` files; and hence modify this. Same below.

Reply:
did so